tests/types/index.ts

```diff
@@ -468,6 +468,16 @@ stripe.confirmGiropayPayment('', {payment_method: ''}, {handleActions: false});
 
 stripe.confirmGiropayPayment('');
 
+stripe.confirmGrabPayPayment('', {return_url: window.location.href});
+
+stripe.confirmGrabPayPayment('');
+
+stripe.confirmGrabPayPayment('', {payment_method: {grabpay: {}}});
+
+stripe.confirmGrabPayPayment('', {payment_method: ''});
+
+stripe.confirmGrabPayPayment('', {payment_method: ''}, {handleActions: false});
```

Comment:
Could you also add some type tests for `stripe.createPaymentMethod` with a GrabPay?

Reply:
Added!